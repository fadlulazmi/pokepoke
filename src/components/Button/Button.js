import React from 'react'
import './styles.scoped.css'

export default function Button({onClick, text}) {
  return (
    <button className="button" onClick={onClick}>{text}</button>
  )
}
