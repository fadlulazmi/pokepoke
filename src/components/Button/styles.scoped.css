.button {
  border-radius: 10px;
  color: chartreuse;
  letter-spacing: 3px;
  background-color: rgba(255, 255, 255, .3);
  cursor: pointer;
}