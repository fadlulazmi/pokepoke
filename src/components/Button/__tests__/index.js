import Button from '../Button'
import ShallowRenderer from 'react-test-renderer/shallow';


describe('src/components/Button', () => {
  test('should be rendered correctly', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Button />);
    expect(tree).toMatchSnapshot();
  })
})
