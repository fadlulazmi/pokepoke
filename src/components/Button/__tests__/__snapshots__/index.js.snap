// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Button should be rendered correctly 1`] = `
<button
  className="button"
/>
`;
